scripts/main.js

```diff
@@ -1,3 +1,36 @@
-define(function() {
-    console.log('App starting.');
+define([
+        'jquery',
+        'lodash',
+        './core/game.js',
+    ], function($, _, Game) {
+        var generateWorld = function() {
+            var width = _.parseInt($('.input-world-width').val());
+            var height = _.parseInt($('.input-world-height').val());
+            var seed = $('.input-world-seed').val();
+
+            if (!_.isNumber(width)
+                || width <= 0
+                || !_.isNumber(height)
+                || height <= 0
+                || seed === ''
+            ) {
+                throw 'You cannot create world without width, height, and seed';
+            }
+
+            var game = new Game(seed, width, height);
+            $('.world-creation-form').addClass('is-hidden');
+
+            // TODO: Pass the game object to the canvas generation part.
```

Comment:
Thats the main JS call from the HTML. Go to the index.html and you ll see the form. When you press generate. it will go here. you can build a class which take the Game object as a parameter and then go call the world and the location for each area and draw the world.